locking/api.go

```diff
@@ -197,6 +197,12 @@ type committer struct {
 	Email string `json:"email"`
 }
 
-func newCommitter(name, email string) committer {
-	return committer{Name: name, Email: email}
+func NewCommitter(name, email string) Committer {
+	return Committer{Name: name, Email: email}
```

Comment:
Should this be returning a `*Committer`, since that's the receiver for `String()` below? Or should the func change the receiver to the struct value?